benchmarks/tools/benchmark.py

```diff
@@ -0,0 +1,176 @@
+from argparse import ArgumentParser
+from collections import OrderedDict
+from collections.abc import Iterable
+from itertools import product
+from datetime import datetime
+from os import path, makedirs
+from devito.logger import warning
```

Comment:
blank line missing before the devito-related imports

Reply:
fixed